### PR TITLE
Support SSL

### PR DESCRIPTION
**Pull request by arnested**

### CODE REVIEW

File: files/etc/apache2/sites-available/ssl.conf [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+		#	 useless for serving static content. So one usually enables the
+		#	 exportation for CGI and SSI requests only.
+		#   o OptRenegotiate:
+		#	 This enables optimized SSL connection renegotiation handling when SSL
+		#	 directives are used in per-directory context.
+		#SSLOptions +FakeBasicAuth +ExportCertData +StrictRequire
+		<FilesMatch "\.(cgi|shtml|phtml|php)$">
+				SSLOptions +StdEnvVars
+		</FilesMatch>
+		<Directory /usr/lib/cgi-bin>
```
Comment (danquah) [inline files/etc/apache2/sites-available/ssl.conf]: We don't use cgi-bin?

File: files/etc/apache2/sites-available/ssl.conf [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+		DocumentRoot /var/www/web
+
+		# Available loglevels: trace8, ..., trace1, debug, info, notice, warn,
+		# error, crit, alert, emerg.
+		# It is also possible to configure the loglevel for particular
+		# modules, e.g.
+		#LogLevel info ssl:warn
+
+		ErrorLog ${APACHE_LOG_DIR}/error.log
```
Comment (danquah) [inline files/etc/apache2/sites-available/ssl.conf:13]: Remember to add a tail of the error-log to etc/service/apache-error-forwarder/run (or introduce an extra service).

(I've avoided tailing the access-log itself for now as developers are mostly interested in errors)

**Review by danquah — APPROVED** [review]
The cgi-bin thing is minor, so I'm ok either way.

**Review by danquah — CHANGES_REQUESTED** [review]
forgot about the log tailer - that should probably be fixed.

File: files/etc/apache2/sites-available/ssl.conf (hunk 2 above)
Comment (arnested) [reply files/etc/apache2/sites-available/ssl.conf:13]: The error log goes to the same file as the error log for port 80 so this should be handled already (although I did not know that -- thanks for enlightening me :smile:)

File: files/etc/apache2/sites-available/ssl.conf (hunk 1 above)
Comment (arnested) [reply files/etc/apache2/sites-available/ssl.conf]: Fixed.

**Review by danquah — APPROVED** [review]

**Pull request merged**